index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Mobile Version</title>
```

Comment:
- [ ] [OPTIONAL]: To make your project more professional, you should consider editing this title, maybe something like "your name's portfolio" would be better?

Reply:
Changed to Vahan's Portfolio.